### DISCUSSION TITLE
Anyway to calculate number of points in an entire column of voxels?

### DISCUSSION THREAD

**Discussion by teide101** [r-lidar/lidR#607] [Q&A] [answered]
Is there anyway to calculate the number of points in an entire column of voxels? I have managed to calculate the number of points in individual voxels (1x1x1m boxes) but I wasn't sure if there was a way to calculate the total number of returns in each column of those 1x1x1m boxes.

**Comment by Jean-Romain** [comment, marked as answer]
The number of point in an entire column of voxel? So you mean the number of point per pixel. Use `pixel_metrics()`
